invenio_files_rest/helpers.py

```diff
@@ -291,3 +292,13 @@ def create_file(key, path):
                 assert filename.startswith(source)
                 parts = [p for p in filename[len(source):].split(os.sep) if p]
                 yield create_file('/'.join(parts), os.path.join(root, name))
+
+
+def create_file_streaming_redirect_response(obj):
```

Comment:
What about if someone has to create a signed url for the final file, with maybe a token per user? Do we have enough params? Does it make sense to do a quick search how other people need this?
E.g. first link on Google:
https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/s3-example-presigned-urls.html

Reply:
As discussed in IRL, who is implementing this function should already have enough objects to access what needed.